src/ProductConstructionService/ProductConstructionService.BarViz/Properties/launchSettings.json

```diff
@@ -0,0 +1,51 @@
+{
+    "$schema": "http://json.schemastore.org/launchsettings.json",
+    "iisSettings": {
+        "windowsAuthentication": false,
+        "anonymousAuthentication": true,
+        "iisExpress": {
+            "applicationUrl": "http://localhost:64328",
+            "sslPort": 44355
+        }
+    },
+    "profiles": {
+        "http": {
```

Comment:
Do we even need the `http` profile?

Reply:
Good point

Reply:
Well, maybe we should just keep this. It is good to have the option and it would be difficult to bring this back

Reply:
Sure, sounds reasonable